class AuthenticationError(Exception):
    pass


class AbortError(Exception):
    pass
